docs/csharp/whats-new/csharp-7.md

```diff
@@ -386,7 +386,7 @@ when reading it the first time. These methods must be understood
 outside of the context of the single calling location.
 
 For those designs, *local functions* enable you to declare methods
-inside the context of another method. This makes it easier for readers
+inside the context of another method or inside constructor. This makes it easier for readers
```

Comment:
I originally chose 'method' because it does explain the concept. Constructors and finalizers *are* methods. 

It does leave off property and event accessors, lambda expressions, and yes, local functions too.

But there's more: you can't declare local functions inside an expression bodied member (or any of the types listed). 

Do either of you have good suggestions? Or, is this pointing to the need for a topic dedicated to local functions?